WISE-Community/WISE
# Added processing indicator when creating run and styled copy project dialog

To test:
- Login as a teacher and view "Browse WISE Units".
- Try to create a run and make sure the "Create Run" button is disabled and the text changes to "Creating..." when clicked. Make sure creating run is successful.
- Try to copy projects and make sure that works.

Closes #1492.
